pkgs/tools/package-management/nix-visualize/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, nix
+, python
+, matplotlib
+, networkx
+, pandas
+, pygraphviz
+}:
+
+python.pkgs.buildPythonApplication rec {
+  version = "1.0.5-unstable-2024-01-17";
+  pname = "nix-visualize";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "craigmbooth";
+    repo = "nix-visualize";
+    rev = "5b9beae330ac940df56433d347494505e2038904";
+    hash = "sha256-VgEsR/Odddc7v6oq2tNcVwCYm08PhiqhZJueuEYCR0o=";
+  };
+
+  postInstall = ''
+    wrapProgram $out/bin/nix-visualize \
+        --prefix PATH : ${lib.makeBinPath [nix]}
+  '';
+
+  propagatedBuildInputs = [
+    matplotlib
+    networkx
+    pandas
+    pygraphviz
+  ];
+
```

Comment:
Missing a pythonImportsCheck, also please disable checkPhase with doCheck = false and add a short comment that there are no tests.

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/299429/commits/32670b1f0a70f04acd77e2cc8e9d40b68c7e1a6c